apps/emqx_bridge_http/test/emqx_bridge_http_v2_SUITE.erl

```diff
@@ -126,84 +126,148 @@ t_compose_connector_url_and_action_path(Config) ->
 %% Checks that we can successfully update a connector containing sensitive headers and
 %% they won't be clobbered by the update.
 t_update_with_sensitive_data(Config) ->
-    ?check_trace(
```

Comment:
`?check_trace` was used to help debug failures, if any, by inspecting the dumped trace.

Reply:
ok, i didn't know that. maybe we can add it back when needed to debug?